HeishaMon/decode.cpp

```diff
@@ -107,8 +107,14 @@ String getModel(byte input) {
       return "8";
     case 51:
       return "9";
+    case 51:
```

Comment:
There's case 51 twice now?

Reply:
indeed, thanks for noticing, just made mistake.